committeeoversightapp/templates/list.html

```diff
@@ -5,10 +5,10 @@
 
  <p><a href="/" class="btn btn-success">Create new hearing</a></p>
 
- <table class="table table-striped" id="hearing-table">
+ <table class="table table-striped table-bordered" id="hearing-table">
   <thead>
     <tr>
-      <th scope="col" class="w-75">Hearing Title</th>
+      <th scope="col" id="hearing-title-col">Hearing Title</th>
```

Comment:
nice - this works, too!

Reply:
I wanted it more precise than the w-*s allowed! A useful trick for the future, though.